.circleci/config.yml

```diff
@@ -1015,6 +1015,11 @@ workflows:
       - binary_linux_wheel:
           conda_docker_image: pytorch/conda-builder:cpu
           cu_version: cpu
+          filters:
+            branches:
+              only: master,nightly
```

Comment:
Can we add `only: *` here to keep this job available on all branches but also make it an opt-in for tags

Reply:
done